### PR TITLE
[Validator] improve and added more Indonesian translation.

### PR DESCRIPTION
**Pull request by IndraGunawan**
| Q             | A
| ------------- | ---
| Branch?       | "2.7"
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

- Improve some translation's grammar.
- Added translation unit no. 73 - 80.

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.id.xlf [hunk 1]
```diff
@@ -36,11 +36,11 @@
             </trans-unit>
             <trans-unit id="9">
                 <source>This field was not expected.</source>
-                <target>Bidang ini tidak diharapkan.</target>
```
Comment (javiereguiluz) [inline src/Symfony/Component/Validator/Resources/translations/validators.id.xlf]: Google Translate says that `Bidang` translates as `field` and `Ruas` as `segment`, so I'd keep the original translation.

**Review by javiereguiluz — APPROVED** [review]
👍 

I left a minor comment, but the rest of the changes are correct according to Google Translate.

File: src/Symfony/Component/Validator/Resources/translations/validators.id.xlf (hunk 1 above)
Comment (IndraGunawan) [inline src/Symfony/Component/Validator/Resources/translations/validators.id.xlf]: hi @javiereguiluz i use the official translation list from government.
you can view here http://www.perpusnas.go.id/assets/uploads/2016/10/Panduan_Pembakuan_Istilah.pdf
the list from national library of Indonesia.

Comment (javiereguiluz) [inline src/Symfony/Component/Validator/Resources/translations/validators.id.xlf]: @IndraGunawan thanks for the reference! The mentioned official translation list is clearly better than the automatic Google translation, so 👍 to those changes too. Thanks!

Comment (IndraGunawan) [inline src/Symfony/Component/Validator/Resources/translations/validators.id.xlf]: the pdf i refer is only for computer terms.
thanks @javiereguiluz

**Comment by fabpot** [comment]
Thank you @IndraGunawan.

**Pull request merged**